Solutions/Authomize/Analytic Rules/Privileged_Machines_Exposed_to_the_Internet.yaml

```diff
@@ -0,0 +1,62 @@
+id: 72891de4-da70-44e4-9984-35fcea98d000
+name: Privileged Machines Exposed to the Internet
+kind: Scheduled
+description: These are AWS Ec2 machines that are exposed to the internet. You can further filter by tags so that you can, for example, find exposed machines that are also "privileged".
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- Discovery
+- Impact
+relevantTechniques:
+- T1580
+query: |-
+  Authomize_v2_CL
+  | extend EventID = id_s
+  | extend Policy = policy_name_s
+  | extend Severity = severity_s
+  | extend Description = description_s
+  | extend Recommendation = recommendation_s
+  | extend URL = url_s
+  | extend Tactics = tactics_s
+  | where Policy contains "Privileged Machines Exposed to the Internet"
+  | project  EventID, Policy, Severity, Description, Recommendation, URL, Category, Tactics
+requiredDataConnectors:
+- connectorId: Authomize
+  dataTypes: [ "Authomize_v2_CL" ]
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: true
+    reopenClosedIncident: false
+    lookbackDuration: 5h
+    matchingMethod: AnyAlert
+    groupByEntities: []
+    groupByAlertDetails: []
+    groupByCustomDetails: []
+eventGroupingSettings:
+  aggregationKind: SingleAlert
+alertDetailsOverride:
+  alertnameFormat: Alert from Authomize - Privileged Machines Exposed to the Internet
+  alertDescriptionFormat: Privileged Machines Exposed to the Internet - These are AWS Ec2 machines that are exposed to the internet. You can further filter by tags so that you can, for example, find exposed machines that are also "privileged".
+  alertDynamicProperties:
+  - alertProperty: AlertLink
+    value: URL
+customDetails:
+  AuthomizeEventID: EventID
+  EventName: Policy
+  EventDescription: Description
+  EventRecommendation: Recommendation
+  ReferencedURL: URL
+suppressionDuration: 5h
+suppressionEnabled: false
+version: 1.0.0
```

Comment:
Please update entity mapping properties:
https://learn.microsoft.com/en-us/azure/sentinel/entities-reference#entity-types-and-identifiers

Reply:
I'm not sure I can use these yet - we need to expose the api in Authomize to enable something like the userAccount details, currently it doesn't provide a name that could be mapped here and be useful so we need to work on exposing and then mapped back here, next iteration and further extending of the incoming data. The next iteration I'll be working on ASIM parser to be able to do this and extending the scripts for pulling more data on the Authomize side.

Reply:
Also this specific event needs a second call to get the machine host names and associated detail which doesn't happen yet - so further details will need to be done to get the specifics from our systems.